langs/f-sharp/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/dotnet/sdk:6.0.403-alpine3.16 as builder
+FROM mcr.microsoft.com/dotnet/sdk:7.0.100-alpine3.16 as builder
```

Comment:
does it work with alpine 3.17? I suspect that'll change the openssl library filenames at least.

Reply:
also 101 is out :-P

Reply:
I just copied this line from the C# file

Reply:
I'll try it.